crates/sui-node/src/lib.rs

```diff
@@ -1680,9 +1683,7 @@ impl SuiNode {
                 let weak_accumulator = Arc::downgrade(&new_accumulator);
                 *accumulator_guard = Some(new_accumulator);
 
-                consensus_epoch_data_remover
-                    .remove_old_data(next_epoch - 1)
-                    .await;
+                consensus_store_pruner.prune(next_epoch).await;
```

Comment:
Note: the semantics now changed and the `next_epoch` is provided (practically the "current" new epoch)

Reply:
This makes sense.